src/AppBundle/Controller/Report/AccountController.php

```diff
@@ -139,25 +141,99 @@ private function fillAccountData(EntityDir\Report\BankAccount $account, array $d
     }
 
     /**
-     * Check bank account has transfers
+     * Check bank account has transactions
      *
      * @param EntityDir\Report\BankAccount $account
      */
-    protected function denyAccessIfAccountHasTransfers(EntityDir\Report\BankAccount $account)
+    protected function denyAccessIfAccountHasTransactions(EntityDir\Report\BankAccount $account)
```

Comment:
is the name still correct considering there are gifts  and transfer checked inside ?

Reply:
Yes it was renamed to reflect a "transaction" rather than a specific money changing action. A transaction in this case just means money leaving or going into an account so would cover gifts, transfers, expenses as well as money in and out.